src/lib/api.js

```diff
@@ -2,6 +2,35 @@ const request = require('request-promise-native');
 
 const archivingApi = require('./archiving-api');
 const environment = require('./environment');
+const log = require('../lib/log');
+
+const logDeprecatedTransitions = (settings) => {
+  const uri = `${environment.instanceUrl}/api/v1/settings/deprecated-transitions`;
+
+  return request({ uri, method: 'GET', json: true})
```

Comment:
I think we should take into account that most installations don't run our latest version of the software, but most app developers do update their local `medic-conf` installation. We have to strive to make medic-conf compatible with old versions of cht-core as well. 
For this reason, I don't think we should hard fail when someone wants to upload config to a version of cht-core that doesn't have this endpoint. 

I think we should catch all 404s and not display errors when they happen, these can be confusing because of their very non-specific message `{"error":"not_found","reason":"Database does not exist."}`.

Reply:
Done!